assets/css/styles.scss

```diff
@@ -12,6 +12,37 @@ body {
   font-style: normal;
 }
 
+.blog {
```

Comment:
Why are all the class names blog? The layout is meant as a default page layout. Name it "content" or "page"